lib/hawkular/operations/operations_api.rb

```diff
@@ -134,7 +134,7 @@ def invoke_specific_operation(operation_payload, operation_name, &callback)
     #
     # @param callback [Block] callback that is run after the operation is done
     def add_deployment(hash, &callback)
-      hash[:enabled] ||= true
+      hash[:enabled] = defined?(hash[:enabled]) ? hash[:enabled] : true
```

Comment:
not sure what is the `defined?` returns in this case, but why not using `hash[:enabled].key?`